src/amo/components/LandingPage/styles.scss

```diff
@@ -36,28 +29,34 @@
 }
 
 .LandingPage-icon {
-  height: 89px;
-  width: 225px;
+  width: 100%;
+  margin: 0;
 
   @include respond-to(medium) {
-    height: 178px;
-    width: 450px;
+    height: 248px;
+    width: 473px;
   }
 }
 
 .LandingPage-addonType-name {
   font-size: $font-size-l;
-  margin: 0;
+  font-weight: lighter;
```

Comment:
We tend to use actual weights over things like `lighter`, any chance you know what this actual value would be and we can use that, just for clarity/consistency?

Reply:
Yep, I'll fix that.